docs/Makefile

```diff
@@ -0,0 +1,180 @@
+# Makefile for Sphinx documentation
```

Comment:
I like to keep the Sphinx default Makefile for all projects. It helps people coming to the project to feel home.